src/errors/helper.js

```diff
@@ -63,7 +63,8 @@ const ERROR_CODES = {
     TWILIO_ERROR: 'TWILIO_ERROR',
     HOOK_ERROR: 'HOOK_ERROR',
     NOT_SUPPORTED: 'NOT_SUPPORTED',
-    ASSERT_PASSED: 'ASSERT_PASSED'
+    ASSERT_PASSED: 'ASSERT_PASSED',
+    CANNOT_FIND: 'CANNOT_FIND'
```

Comment:
This should be changed to `MODULE_NOT_FOUND`, for the sake of consistency with other error names.

Reply:
fixed